From: Chuan-kai Lin <cklin@debian.org>
Date: Sun, 16 Jun 2019 13:12:30 -0700
Subject: Keep build stamp files

Do not delete stamp files outside the Make rules that generates those
same stamp files.  Force deletion of stamp files interferes with our
ability to prevent example extraction from non-DFSG documentation source
(moved into the bison-doc source package) and also breaks parallel
builds.

Bug-Debian: http://bugs.debian.org/732034
Forwarded: not-needed
Last-Update: 2019-06-16
---
 examples/local.mk | 1 -
 1 file changed, 1 deletion(-)

diff --git a/examples/local.mk b/examples/local.mk
index 51c2c5a..414e0c2 100644
--- a/examples/local.mk
+++ b/examples/local.mk
@@ -62,7 +62,6 @@ MAINTAINERCLEANFILES += $(extracted) %D%/extracted.stamp
 	$(AM_V_at)mv $@.tmp $@
 
 $(extracted): %D%/extracted.stamp
-	@test -f $@ || rm -f %D%/extracted.stamp
 	@test -f $@ || $(MAKE) $(AM_MAKEFLAGS) %D%/extracted.stamp
 
 
